app/helpers/feature_flag_helper.rb

```diff
@@ -0,0 +1,15 @@
+module FeatureFlagHelper
+    def self.enabled?(feature)
+        case feature.to_sym
+
+        when :on_sandbox
+            if Rails.application.secrets.on_sandbox.to_s == 'true'
+                true
+            else
+                false
+            end
```

Comment:
This could be refactored to avoid using the if statement it coud just  be

```ruby
Rails.application.secrets.on_sandbox.to_s == 'true'
```

but we can refactor at a later time

Reply:
oh that's cool! I'll do the refactor in another ticket later with the Rspec test